packages/x-components/src/x-modules/facets/components/filters/simple-filter.vue

```diff
@@ -2,7 +2,7 @@
   <RenderlessFilter
     v-slot="{ filter, clickFilter, cssClasses, isDisabled }"
     :class="cssClasses"
-    :clickEvents="_clickEvents"
+    :clickEvents="clickEventsEmit"
```

Comment:
This is not having any effect on the Vue3 playground. Classes added by this component are not applied.